web/packages/shared/components/UnifiedResources/UnifiedResources.tsx

```diff
@@ -355,10 +359,27 @@ export function UnifiedResources(props: UnifiedResourcesProps) {
     unifiedResourcePreferences.labelsViewMode ===
     LabelsViewMode.LABELS_VIEW_MODE_EXPANDED;
 
+  useEffect(() => {
+    const resizeObserver = new ResizeObserver(entries => {
+      const container = entries[0];
+      if (container.contentRect.width <= 800) {
```

Comment:
I'd extract that `800` value to a global constant and move to the top of the file.